lib/auth/clt.go

```diff
@@ -490,26 +490,6 @@ func (c *Client) DeactivateCertAuthority(id types.CertAuthID) error {
 	return trace.NotImplemented(notImplementedMessage)
 }
 
-// GenerateToken creates a special provisioning token for a new SSH server
-// that is valid for ttl period seconds.
-//
-// This token is used by SSH server to authenticate with Auth server
-// and get signed certificate and private key from the auth server.
-//
-// If token is not supplied, it will be auto generated and returned.
-// If TTL is not supplied, token will be valid until removed.
-func (c *Client) GenerateToken(ctx context.Context, req GenerateTokenRequest) (string, error) {
-	out, err := c.PostJSON(ctx, c.Endpoint("tokens"), req)
-	if err != nil {
-		return "", trace.Wrap(err)
-	}
-	var token string
-	if err := json.Unmarshal(out.Bytes(), &token); err != nil {
-		return "", trace.Wrap(err)
-	}
-	return token, nil
-}
-
```

Comment:
Is this being replaced with `api/client.GenerateToken`? This is not a nit, I'm just trying to get a better understanding of this change. ;)

Edit: Ah no, this was moved to httpfallback.go.

Reply:
Yeah it's replaced by the method in `httpfallback.go`, and then once we remove that in 11.0.0, it will be replaced by `api/client.GenerateToken`.